core/src/main/java/org/elasticsearch/action/search/MultiSearchResponse.java

```diff
@@ -111,11 +112,14 @@ public Exception getFailure() {
     }
 
     private Item[] items;
+    
+    private long tookInMillis;
 
     MultiSearchResponse() {
     }
 
-    public MultiSearchResponse(Item[] items) {
+    public MultiSearchResponse(long tookInMillis, Item[] items) {
```

Comment:
Let's flip the order of these parameters (in general, I think that new parameters should be added at the end absent a good reason otherwise, but putting it at the end also aligns it more closely with the `BulkResponse`; it wouldn't surprise me we are not consistent about this through the codebase, but I'm choosing the `BulkResponse` here to be consistent with).